### PR TITLE
Set `EnforcedStyle: indented` for Two `Layout/Multiline*Indentation` Rules

### PR DESCRIPTION
**Pull request by Hamms**
Specifically:

`Layout/MultilineMethodCallIndentation`:
> Checks the indentation of the method name part in method calls that span more than one line.

`Layout/MultilineOperationIndentation`:
> Checks the indentation of the right hand side operand in binary operations that span more than one line.

Both rules largely work the same, but apply in different circumstances; handling them at the same time helps keep our codebase consistent.

In both cases, we have the option to either use the `indented` style that this PR proposes or the default style, but I think `indented` is a lot cleaner. See [this PR](https://github.com/code-dot-org/code-dot-org/pull/53644) for an example of what applying the default style rules to our codebase would look like, and more of my thoughts on why it's not an ideal option. 

We also have the option of intentionally disabling this rule and allowing developers to continue to apply whatever indentation style they'd prefer.

What does everyone think?

## Links

- https://www.rubydoc.info/gems/rubocop/RuboCop/Cop/Layout/MultilineMethodCallIndentation
- https://www.rubydoc.info/gems/rubocop/RuboCop/Cop/Layout/MultilineOperationIndentation
- https://docs.rubocop.org/rubocop/cops_layout.html#layoutmultilinemethodcallindentation
- https://docs.rubocop.org/rubocop/cops_layout.html#layoutmultilineoperationindentation

### CODE REVIEW

**Review by snickell — APPROVED** [review]
I think your instincts are good, I'm also preferring this style. The deep and varied indents of the other "default" style strike me as hard to follow... visually just end up with random indents all over the place, rather than a fixed "gutter depth" to look at. My 2c.

**Comment by Hamms** [comment]
My thoughts exactly; thank you!
